<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_App</class>
 <widget class="QMainWindow" name="Main_App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SEC4_test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>File name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name_lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="file_name_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="message_textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>251</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="message_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>89</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="list_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>List of inputs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>427</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
